### PR TITLE
Refactor MicroProfile Metrics 2.0 CDI bundles 

### LINKED ISSUE #7781: [19.0.0.7] Refactor metrics CDI packages to a common Bundle
Common classes/packages used in com.ibm.ws.microprofile.metrics.*.cdi should refactored to a common bundle.

### PR DESCRIPTION
**Pull request by Channyboy**
fixes #7781 
#build

### CODE REVIEW

**Comment by Channyboy** [comment]
#build

**Comment by Channyboy** [comment]
#build

File: dev/com.ibm.ws.microprofile.metrics.cdi.2.0/src/io/astefanutti/metrics/cdi20/MetricName.java [hunk 1]
```diff
@@ -118,21 +109,18 @@ private String getParameterName(AnnotatedParameter<?> parameter) {
     }
 
     /** {@inheritDoc} */
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics.cdi.2.0/src/io/astefanutti/metrics/cdi20/MetricName.java]: Should be removed as the class does not implement `MetricName` anymore.

File: dev/com.ibm.ws.microprofile.metrics.cdi.2.0/src/io/astefanutti/metrics/cdi20/MetricName.java [hunk 2]
```diff
@@ -118,21 +109,18 @@ private String getParameterName(AnnotatedParameter<?> parameter) {
...
         return metadataOf(annotated, name, type);
     }
 
-    @Override
     public String[] tagOf(InjectionPoint ip) {
         Annotated annotated = ip.getAnnotated();
         return tagOf(annotated);
     }
 
     /** {@inheritDoc} */
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics.cdi.2.0/src/io/astefanutti/metrics/cdi20/MetricName.java]: Should be removed

File: dev/com.ibm.ws.microprofile.metrics.cdi.2.0/src/io/astefanutti/metrics/cdi20/MetricName.java [hunk 3]
```diff
@@ -188,13 +176,11 @@ private String metadataValueOf(String value) {
...
```
Comment (fmhwong) [inline dev/com.ibm.ws.microprofile.metrics.cdi.2.0/src/io/astefanutti/metrics/cdi20/MetricName.java]: Should be removed

**Comment by Channyboy** [comment]
#run-libby-bot

**Comment by Channyboy** [comment]
#build

**Pull request merged**